### PR TITLE
Feature : Show 10 notification history in tooltip

### PR DESCRIPTION
**Pull request by mahaveergurjar**
# Pull Request

## Description
Show notification  history  uptop 10 notifications

## Type of change

Please put an `x` in the boxes that apply:

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [x] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [ ] **Documentation update** (non-breaking change; modified files are limited to the documentations)
- [ ] **Technical debt** (a code change that does not fix a bug or add a feature but makes something clearer for devs)
- [ ] **Other** (provide details below)

## Checklist

Please put an `x` in the boxes that apply:

- [x] I have read the [CONTRIBUTING](https://github.com/prasanthrangan/hyprdots/blob/main/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My commit message follows the [commit guidelines](https://github.com/prasanthrangan/hyprdots/blob/main/CONTRIBUTING.md#git-commit-messages).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added a changelog entry.
- [x] I have added necessary comments/documentation to my code.
- [x] I have added tests to cover my changes.
- [x] I have tested my code locally and it works as expected.
- [ ] All new and existing tests passed.

## Screenshots
240722_19h47m24s_screenshot
image

(if appropriate)

### CODE REVIEW

**Comment by mahaveergurjar** [comment]
@kRHYME7 review please this pr

**Comment by kRHYME7** [comment]
Can you try to add maybe bullet? for each notifs

**Comment by mahaveergurjar** [comment]
ok

**Comment by mahaveergurjar** [comment]
image
image
image
which on is add ???

**Comment by kRHYME7** [comment]
1st I guess. Just make it simple. Tnx

**Comment by mahaveergurjar** [comment]
ok
I add

**Comment by mahaveergurjar** [comment]
@kRHYME7 know test added all things

**Comment by mahaveergurjar** [comment]
@kRHYME7 please review this pr

File: Configs/.local/share/bin/notifications.py [hunk 1]
```diff
@@ -15,25 +14,24 @@ def format_history(history):
     alt = 'none'
     tooltip_click = []
     tooltip_click.append("󰎟 Notifications")
-    tooltip_click.append("󰳽 click-left:  history pop")
```
Comment (kRHYME7) [inline Configs/.local/share/bin/notifications.py:18]: Can you also add this line 
    
``` tooltip_click.append("󰳽 scroll-down:  history pop") ```

**Review by kRHYME7 — CHANGES_REQUESTED** [review]

**Comment by mahaveergurjar** [comment]
Ok

**Comment by mahaveergurjar** [comment]
Added

**Comment by mahaveergurjar** [comment]
@kRHYME7 please review this pr

**Comment by hdengsan** [comment]
What about using the sway notification center and just theme it. It's already a dependency. It will handle this better than some bash scripts.

**Review by Sitolam — APPROVED** [review]

**Review by kRHYME7 — APPROVED** [review]
Thanks!

File: Configs/.local/share/bin/notifications.py [hunk 2]
```diff
@@ -15,25 +14,26 @@ def format_history(history):
...
+    tooltip_click.append("󰳽 scroll-down:  history pop")
```
[the remark this answers is not in the document]
Comment (mahaveergurjar) [reply Configs/.local/share/bin/notifications.py:17]: I removed it but I  think it does not remove

[the remark this answers is not in the document]
Comment (kRHYME7) [reply Configs/.local/share/bin/notifications.py:17]: I think it's fine now?

**Pull request merged**